src/inc/msquic.h

```diff
@@ -635,7 +636,8 @@ typedef struct QUIC_SETTINGS {
     uint8_t MigrationEnabled                : 1;
     uint8_t DatagramReceiveEnabled          : 1;
     uint8_t ServerResumptionLevel           : 2;    // QUIC_SERVER_RESUMPTION_LEVEL
-    uint8_t RESERVED                        : 2;
+    uint8_t GreaseQuicBitEnabled            : 1;
```

Comment:
We should probably expose whether this was negotiated or not in the QUIC statistics. Also, we should update the QUIC interop tool to enable this.

Reply:
Alright! I'm gonna add some stat fields about it. Also gonna take a look at QUIC Interop tools as well!

Reply:
I added stats, and I'll add them to the necessary places to increase values.